lib/assert.js

```diff
@@ -83,16 +83,14 @@ function ok(value, message) {
 assert.ok = ok;
 
 // The equality assertion tests shallow, coercive equality with ==.
-/* eslint-disable no-restricted-properties */
+/* eslint-disable no-restricted-properties, eqeqeq */
```

Comment:
Non-blocking comment, but I'd rather we kept the `eqeqeq` disable scoped to a single line as it is in the current code.

Reply:
I reverted that change again.